bindings/netstandard/ElectionGuard/ElectionGuard.Shared/Encrypt.cs

```diff
@@ -22,18 +22,16 @@ public unsafe EncryptionDevice(ulong uuid, string location)
         protected override unsafe void DisposeUnmanaged()
         {
             base.DisposeUnmanaged();
-            var status = NativeInterface.EncryptionDevice.Free(Handle);
-            if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
-            {
-                Console.WriteLine($"DisposeUnmanaged Error Status: {status}");
-            }
+
+            if (Handle == null || Handle.IsInvalid) return;
```

Comment:
Could this be a static method? `Dispose(Handle)` or something similar that would set the Handle to null at the end.